config/samples/itz-bund/easyredmine.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: k8s.cloudogu.com/v1
+kind: Dogu
+metadata:
+  name: easyredmine
+  annotations:
+    test: dev
+  labels:
+    dogu.name: easyredmine
```

Comment:
Did I enter `dogu.name: x` labels during the label creation story? If so I might have added a false cue. IIRC `dogu.name` is used for pods to uniquely identify different pod versions of the same dogu. 

Maybe we should remove these labels (unless they are actively used) because a dogu can be uniquely addressed by its very own name.

Reply:
These labels have been deleted along with those unnecessary annotations.